### DISCUSSION TITLE
Osu! Lazer doesn't open after updating it on Linux

### DISCUSSION THREAD

**Discussion by Ambiance3757** [ppy/osu#30709] [Q&A] [answered]
After updating the client to the latest version it no longer opens due to a problem with the libssl library.

Log: `/tmp/.mount_osu.ApA8QIHf/usr/bin/UpdateNix: error while loading shared libraries: libssl.so.1.0.0: cannot open shared object file: No such file or directory
Unhandled exception. System.Exception: Update.exe process exited too soon (127).
   at Velopack.UpdateExe.Apply(IVelopackLocator locator, VelopackAsset toApply, Boolean silent, Boolean restart, String[] restartArgs, ILogger logger)
   at Velopack.VelopackApp.Run(ILogger logger)
   at osu.Desktop.Program.setupVelopack()
   at osu.Desktop.Program.Main(String[] args)
Abortado ('core' generado)`

I checked what version of libssl my system has and it's libssl.so.1.1, so that would explain the error, but how to I fix it?

System:
OS: Arch Linux x86_64 
Kernel: 6.11.8-arch1-2 
Using Wayland.

**Comment by felipemarins** [comment, marked as answer]
Try to download the latest appimage manually and launch it.

**Reply by Ambiance3757** [reply]
Thanks, that fixed the problem! I wonder why did it happen though; it might be a problem with the updating system within the client, or something like that.

**Comment by skibon02** [comment]
The same issue. Manual re-download from github releases helps
